e2e/pages/adf/viewerPage.ts

```diff
@@ -185,6 +186,10 @@ export class ViewerPage {
         Util.waitUntilElementIsVisible(this.closeButton);
     }
 
+    getLastButtonDisplayed() {
```

Comment:
I would rename this method to getLastButtonTitle() and return in here the value of the 'title' attribute and just use it in the test. Same for moreActionsMenu.

Reply:
done, thanks